### DISCUSSION TITLE
I want try "MAXIMUS-VII-RANGER-ASUS-3503-REBAR" in my system, i thinks it's similar MB and bios

### DISCUSSION THREAD

**Discussion by patfun69** [xCuri0/ReBarUEFI#154] [General]
**System**

 - SO: Microsoft Windows 11 Professional (x64) Build 22631.3085 (23H2)
 - Motherboard: Asus Z97 PRO (WIFI AC USB3.1) 
 - CHIP: Intel Core i7-4790K
 - BIOS Version: 3503 Beta (2018)
 - GPU: Gigabyte Winforce 3 GTX 1080
 - RAM: Corsair 32 GB DDR3-2400/PC3-19200 CMY32GX3M4A22400C11
 
 - [X] CSM is turned off. (BIOS)
 - [X] 4G decoding is enabled. (BIOS)

The first metod "ReBarUEFI-master" dont function for me. I would like to be able to use this ability also on my old MB with the prospect of mounting a new GPU, such as RTX 4060Ti. The first method doesn't work, and moving through the various posts to draw conclusions is not easy. Maybe "MAXIMUS-VII-RANGER-ASUS-3503-REBAR" comes close to my solution? Any suggestions? Thanks.

**Comment by PatricLeal** [comment]
I have an Asus Z97 Pro Wifi AC running this Bios 3503 together with an RTX 3060. Basically, the only thing missing is activating the bios, could anyone tell me how to do it in this bios.
Attached below is the bios I use and system data.

[Z97-PRO-WIFI-AC-USB31-ASUS-3503.zip](https://github.com/xCuri0/ReBarUEFI/files/14531529/Z97-PRO-WIFI-AC-USB31-ASUS-3503.zip)

![image](https://github.com/xCuri0/ReBarUEFI/assets/26906375/72e7b80e-2d50-4407-b6d1-3d031d008994)
